src/components/AvatarGroup/AvatarGroupCounterTooltipContent.jsx

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import Clickable from "../Clickable/Clickable";
+import Flex from "../Flex/Flex";
+import Avatar from "../Avatar/Avatar";
+import cx from "classnames";
+import PropTypes from "prop-types";
+import styles from "./AvatarGroupCounterTooltipContent.module.scss";
+
+const AvatarGroupCounterTooltipContent = ({ avatars, type, className }) => {
+  const getTooltipContent = avatarProps => {
+    return avatarProps?.tooltipProps?.content || avatarProps?.ariaLabel;
+  };
+
+  avatars = avatars.map(avatar => ({ value: avatar.props }));
+  const displayAsGrid = !avatars.some(item => getTooltipContent(item.value));
```

Comment:
why to display as grid?
some logic should be under use memo

Reply:
If there is no tooltip text we display avatars as grid - Evgeniy asked for it
![image](https://user-images.githubusercontent.com/104433616/174056599-c7675e7f-b95e-4759-8966-de291f2d08d4.png)
I'll use memo here